### PR TITLE
{bio}[GCCcore/10.2.0] vawk v0.0.1 w/ Python 2.7.18

### PR DESCRIPTION
**Pull request by scicomp-moffitt**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/vawk/vawk-0.0.1-GCCcore-10.2.0-Python-2.7.18.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'vawk'
+version = '0.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cc2qe/vawk'
+description = """An awk-like VCF parser 
+"""
+
+toolchain =  {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/v/vawk/vawk-0.0.1-GCCcore-10.2.0-Python-2.7.18.eb:12]:
```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
This should fix:
```
vawk-0.0.1-GCCcore-10.2.0-Python-2.7.18.eb:12:12: E222 multiple spaces after operator
```

File: easybuild/easyconfigs/v/vawk/vawk-0.0.1-GCCcore-10.2.0-Python-2.7.18.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'vawk'
+version = '0.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cc2qe/vawk'
+description = """An awk-like VCF parser 
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/v/vawk/vawk-0.0.1-GCCcore-10.2.0-Python-2.7.18.eb:12]:
```suggestion
toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
I think this should go in the `GCC` toolchain. `GCC` contains `GCCcore` and `binutils` which is needed for `Python`.

**Comment by scicomp-moffitt** [comment]
Currently I always wait until someone explains the error message from the bot. Is there any way I can find that myself ? Where do these "E216 ...blala" explanations come from? I cannot find them .

**Comment by smoors** [comment]
@scicomp-moffitt as discussed with @SebastianAchilles we can keep the `GCCcore` toolchain and add `binutils` as build dep.
I just made these changes for you.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @scicomp-moffitt!

**Pull request merged**